spring-integration-core/src/test/java/org/springframework/integration/handler/ServiceActivatorDefaultFrameworkMethodTests.java

```diff
@@ -221,4 +277,19 @@ public String processMessage(Message<?> message) {
 		}
 	}
 
+	private static class AsyncService {
```

Comment:
We can add `RedundantModifier` to the `checkstyle-suppressions.xml` for the tests, but maybe it would be better to keep similar classes as `public`?
E.g. I fixed several of them recently because of automatic `fixModifiers`.